topics/scikit-learn/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: sklearn
+created_by: David @cournape Cournapeau, The scikit-learn developers
+display_name: scikit-learn
+github_url: https://github.com/scikit-learn/scikit-learn
+released: January 05, 2010
+short_description: scikit-learn is a Python module for machine learning.
+topic: scikit-learn
+related: scikit, python
+url: https://github.com/scikit-learn/scikit-learn/releases
+wikipedia_url: https://en.wikipedia.org/wiki/Scikit-learn
+---
+scikit-learn is a widely used de-facto standard Python module for classic machine learning built on top of SciPy.
```

Comment:
_"de-facto standard"_ seems redundant, could you remove `de-facto` here?

Reply:
It's not standardized, it's just very popular. See https://en.wikipedia.org/wiki/De_facto_standard for more info.

Reply:
@KOLANICH Please make the change as requested, thanks!

Reply:
can remove `de-facto standard` and just say "widely used", that should be sufficient.